pkg/shell/base_index.js

```diff
@@ -93,8 +93,10 @@ function Frames(index, setupIdleResetTimers) {
             if (frame.contentWindow && setupIdleResetTimers)
                 setupIdleResetTimers(frame.contentWindow);
 
-            if (frame.contentDocument && frame.contentDocument.documentElement)
+            if (frame.contentDocument && frame.contentDocument.documentElement) {
                 frame.contentDocument.documentElement.lang = language;
+                frame.contentDocument.documentElement.dir = cockpit.language_direction;
```

Comment:
Do we need to be concerned about pages with an older cockpit.js?  Remember that c-podman/machines and also e.g. the OpenSUSE installers bundle cockpit.js into their webpacks. Let's rather only set this if `cockpit.language_direction` is truthy. (Same in indexes.jsx).  With that we can also simplify the code above (but it's also ok as -is)

Or is it ok to set `.dir` to undefined?